recipe/meta.yaml

```diff
@@ -1,17 +1,16 @@
-{% set version = "4.11.1" %}
+{% set version = "4.12.0" %}
 
 package:
-  name: beautifulsoup4
+  name: beautifulsoup4-split
```

Comment:
can you explain why we change the name ?

Reply:
The linter says: `ERROR: /tmp/abs_071e1cwsmh/clone/recipe/meta.yaml:16: outputs_not_unique: output "beautifulsoup4": Output name is not unique`

Reply:
https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/Reference/Recipe_standards.md#the-package-section

Reply:
find ./ -name meta.yaml -exec grep "name: beautifulsoup4" {} +
./beautifulsoup4-feedstock/recipe/meta.yaml:  name: beautifulsoup4
./beautifulsoup4-feedstock/recipe/meta.yaml:  - name: beautifulsoup4

the second found is from outputs